app/ante/min_fee_test.go

```diff
@@ -100,11 +100,19 @@ func TestCheckTxFeeWithGlobalMinGasPrices(t *testing.T) {
 					App: tc.appVersion,
 				},
 			})
-			_, _, err := ante.CheckTxFeeWithGlobalMinGasPrices(ctx, tx)
+
+			_, _, globalMinGasPriceErr := ante.CheckTxFeeWithGlobalMinGasPrices(ctx, tx)
+			// test that the fee checking logic produces the same result as in CheckTxFeeWithMinGasPrices
+			minGasPriceErr := ante.CheckTxFeeWithMinGasPrices(tx, appconsts.GlobalMinGasPrice, "insufficient global minimum fee")
+
 			if tc.expErr {
-				require.Error(t, err)
+				require.Error(t, globalMinGasPriceErr)
+				require.Error(t, minGasPriceErr)
+				require.Equal(t, globalMinGasPriceErr.Error(), minGasPriceErr.Error())
 			} else {
-				require.NoError(t, err)
+				require.NoError(t, globalMinGasPriceErr)
+				require.NoError(t, minGasPriceErr)
+				require.Equal(t, globalMinGasPriceErr, minGasPriceErr)
```

Comment:
[optional] do we need this assertion if both errors are `nil`? I would never expect this line to catch something that wasn't caught by the previous two lines 🤔 

```suggestion
```